classes/mail/variables/ReviewAssignmentEmailVariable.inc.php

```diff
@@ -52,29 +60,40 @@ public function values(string $locale): array
     {
         return
         [
-            self::REVIEW_DUE_DATE => $this->getReviewDueDate(),
-            self::RESPONSE_DUE_DATE => $this->getResponseDueDate(),
-            self::SUBMISSION_REVIEW_URL => $this->getSubmissionUrl(),
+            self::REVIEW_DUE_DATE => $this->getReviewDueDate($locale),
+            self::RESPONSE_DUE_DATE => $this->getResponseDueDate($locale),
+            self::REVIEW_ASSIGNMENT_URL => $this->getSubmissionUrl(),
         ];
     }
 
-    protected function getReviewDueDate() : string
+    protected function getReviewDueDate($locale): string
     {
-        return $this->reviewAssignment->getDateDue();
+        $reviewDueDate = strtotime($this->reviewAssignment->getDateDue());
+        if ($reviewDueDate === -1 || $reviewDueDate === false) {
+            // Default to something human-readable if no date specified
+            return '_____';
+        }
+        return strftime($this->context->getLocalizedDateFormatShort($locale), $reviewDueDate);
     }
 
-    protected function getResponseDueDate() : string
+    protected function getResponseDueDate($locale): string
     {
-        return $this->reviewAssignment->getDateResponseDue();
+        $responseDueDate = strtotime($this->reviewAssignment->getDateResponseDue());
+        if ($responseDueDate === -1 || $responseDueDate === false) {
+            // Default to something human-readable if no date specified
+            return '_____';
```

Comment:
Let's return `'{$' . self::RESPONSE_DUE_DATE . '}`. That will make it look like an incomplete variable, but also give people some clue as to what it might mean.

(same above)